recipe/meta.yaml

```diff
@@ -43,6 +44,8 @@ requirements:
     - jsondiff 1.1.1
     - docker-py
     - backports.tempfile
+    - idna >=2.5,<2.9
+    - cfnlint
```

Comment:
I was just looking at this and came up with a slightly different changeset: 

     - python
-    - jinja2 >=2.7.3
+    - jinja2 >=2.10.1
     - boto >=2.36.0
-    - botocore >=1.12.13
-    - boto3 >=1.6.15
-    - cookies
+    - boto3 >=1.9.86
+    - botocore >=1.12.86
+    - cryptography >=2.3.0
     - requests >=2.5
     - xmltodict
-    - dicttoxml
-    - six
+    - six >1.9
     - werkzeug
-    - pyaml
+    - pyyaml
     - pytz
-    - python-dateutil
+    - python-dateutil >=2.1,<3.0.0
+    - python-jose <4.0.0
     - mock
-    - flask
-    - responses >=0.9.0
-    - python-jose <3.0.0
+    - docker-py >=2.5.1
+    - jsondiff 1.1.2
     - aws-xray-sdk !=0.96,>=0.93
-    - jsondiff 1.1.1
-    - docker-py
-    - backports.tempfile
+    - responses >=0.9.0
+    - idn >=2.5,<2.9
+    - cfn-lint
+    - flask

Reply:
copyable run section in case you wanted to update to this

```
  run:
    - python
    - jinja2 >=2.10.1
    - boto >=2.36.0
    - boto3 >=1.9.86
    - botocore >=1.12.86
    - cryptography >=2.3.0
    - requests >=2.5
    - xmltodict
    - six >1.9
    - werkzeug
    - pyyaml
    - pytz
    - python-dateutil >=2.1,<3.0.0
    - python-jose <4.0.0
    - mock
    - docker-py >=2.5.1
    - jsondiff 1.1.2
    - aws-xray-sdk !=0.96,>=0.93
    - responses >=0.9.0
    - idn >=2.5,<2.9
    - cfn-lint
    - flask
```